src/constants/skills.js

```diff
@@ -410,7 +423,16 @@ const icons = {
   vuepress:
     "https://raw.githubusercontent.com/AliasIO/wappalyzer/master/src/drivers/webextension/images/icons/VuePress.svg",
   unity:
-    "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg"
+    "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg",
+  JAM_stack: 
+    "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/jam.svg",
+  MEAN_stack:
+   "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/mean.svg",
+  MERN_stack:
+    "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/mern.svg",
```

Comment:
```suggestion
    "https://upload.wikimedia.org/wikipedia/commons/9/94/MERN-logo.png",
```

Reply:
hey i guess only svg are allowed !

Reply:
@sixthcodebrewer Ahh my bad, better to commit in the logo as well I feel.